sql-ballerina/Config.toml

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+# Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
We do not change license year for every commit. This is added only in the first commit

Reply:
Actually, I removed the `ballerina.conf` file and added new one with `Config.toml` with updated license. 
But, github shows as a renamed file.
Shall we continue with this change, since we changed the both the name & file extension

Reply:
Ack